src/main/java/com/xmslyz/luhn/logic/LuhnChecksumComputing.java

```diff
@@ -2,57 +2,49 @@
 
 import java.math.BigInteger;
 
-public class LuhnChecksumComputing implements Computation {
+public class LuhnChecksumComputing implements Computable {
 
-    String number, checkSum;
+    int total, checkedNumeral, digitLength, arrInt;
```

Comment:
Pole `total` może zostać zastąpione w nast. sposób:
1. metoda `countTotal` zwraca wartość, tj
```java
private int countTotal() {
    int total = 0;
    for (String s : array) {
        arrInt = Integer.parseInt(s);
        total += arrInt;
    }
    return total;
}
```
2. Przypisujemy wynik tej metody do zmiennej w metodzie `compute`  i przekazujemy go do `getChecksum()`
```java
int total =  countTotal();
return getChecksum(total); //tu też można zwrócić typ `String` zamiast ustawiać wartość pola w klasie.
```
Podobnie można pozbyć się pozostałych pól w klasie. Jest to zalecane ze względu na bezpieczeństwo wielowątkowe (zbyt zaawansowany temat póki co, ale warto się trzymać tej zasady)